### PR TITLE
Fix #2513: Upgrade Kubernetes Model to v1.19.1

### LINKED ISSUE #2513: Add Support for Kubernetes 1.19.0
Kubernetes 1.19.0 got released a month ago. we should upgrade our Kubernetes Model to 1.19.0

https://github.com/kubernetes/kubernetes/releases/tag/v1.19.0
https://kubernetes.io/blog/2020/08/26/kubernetes-release-1.19-accentuate-the-paw-sitive/

### PR DESCRIPTION
**Pull request by rohanKanojia**
Fix #2513 

Updated Kubernetes Model to v1.19.1

- Introduced another module `kubernetes-model-node` for `node.k8s.io/v1beta1` RuntimeClass
- Added `networking.k8s.io/v1` Ingress
- Added `networking.k8s.io/v1` IngressClass

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [X] Breaking change (fix or feature that would cause existing functionality to change
 - [X] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [X] Code contributed by me aligns with current project license: [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
 - [X] I Added [CHANGELOG](../CHANGELOG.md) entry regarding this change
 - [X] I have implemented unit tests to cover my changes
 - [X] I have added/updated the [javadocs](https://www.javadoc.io/doc/io.fabric8/kubernetes-client/latest/index.html) and other [documentation](https://github.com/fabric8io/kubernetes-client/blob/master/doc/CHEATSHEET.md) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=fabric8io_kubernetes-client) report
 - [X] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +19,8 @@
 _**Note**_: Breaking changes in the API
 * Tekton Pipeline v1beta1 *sidecars* field from a task step is now a list of `io.fabric8.tekton.pipeline.v1beta1.Sidecar` instead of a list of `io.fabric8.tekton.pipeline.v1beta1.Step`
 
+* `NetworkPolicy` moved to `io.fabric8.kubernetes.api.model.networking.v1.NetworkPolicy` from
+  `io.fabric8.kubernetes.api.model.networking.NetworkPolicy`
```
Comment (manusa) [inline CHANGELOG.md:23]: So we no longer have access to `extensions/v1beta1` `NetworkPolicy`? Or is this provided by `V1beta1NetworkAPIGroupDSL v1beta1()` DSL method?

File: .github/workflows/e2e-tests.yml [hunk 2]
```diff
@@ -35,14 +35,14 @@ jobs:
...
-        kubernetes: [v1.18.6,v1.12.0]
+        kubernetes: [v1.19.2,v1.18.6,v1.13.1]
     steps:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Setup Minikube-Kubernetes
         uses: manusa/actions-setup-minikube@v2.0.1
         with:
-          minikube version: v1.12.1
+          minikube version: v1.13.1
```
Comment (manusa) [inline .github/workflows/e2e-tests.yml:45]: Why?

Comment (rohanKanojia) [reply .github/workflows/e2e-tests.yml:45]: I wanted to add integration tests for Kubernetes `1.19.x`

Comment (manusa) [reply .github/workflows/e2e-tests.yml:45]: Sorry, comment in wrong place. I'm talking about he upgrade to v1.13.1 if the minimum k8s version.

Comment (rohanKanojia) [reply .github/workflows/e2e-tests.yml:45]: I checked and seems like Kubernetes version 1.12.1 is less than the oldest supported version: v1.13.0 for minikube v1.13.1

Comment (manusa) [reply .github/workflows/e2e-tests.yml:45]: If tests pass we should keep the older version

File: kubernetes-itests/src/test/java/io/fabric8/commons/ClusterEntity.java [hunk 3]
```diff
@@ -49,6 +50,17 @@ public static void remove(InputStream inputStream) {
     }
   }
 
+  public static boolean kubernetesVersionAtLeast(String majorVersion, String minorVersion) {
```
Comment (manusa) [inline kubernetes-itests/src/test/java/io/fabric8/commons/ClusterEntity.java:53]: One last comment:
I'm not sure this method behaves as expected.
I suspect the output of `kubernetesVersionAtLeast("1", "19")` is false when checking a `2.0.0`/`2.0` cluster, which is not the expected output.

**Review by manusa — APPROVED** [review]

**Comment by manusa** [comment]
[merge]

**Comment by rohanKanojia** [comment]
[merge]

**Pull request merged**